cmd/utils/flaggroup.go

```diff
@@ -118,6 +118,26 @@ var FlagGroups = []FlagGroup{
 			MaxBlockDiffFlag,
 		},
 	},
+	{
+		Name: "CHAINDATAFETCHER",
```

Comment:
```suggestion
		Name: "CHAINDATA FETCHER",
```
How about this?

Reply:
Hmm... It is like `Servicechain` instead of `Service chain`. Kind a composite word?

Reply:
Okay :)